tunnel/intra/doh/padding.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2019 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package doh
+
+import (
+	"golang.org/x/net/dns/dnsmessage"
+)
+
+const (
+	OptResourcePaddingCode = 12
+	OptDefaultPaddingLen   = 400
```

Comment:
Let's stick with the RFC 8467 recommendation of 128.  Also, this is more of a "block size" than a "padding length".